From 4af6e8dc7cd7f4f0aba843b47a613864b720ed01 Mon Sep 17 00:00:00 2001
From: "Marco A. Gutierrez" <marco@openrobotics.org>
Date: Tue, 21 Jul 2020 12:38:39 +0800
Subject: [PATCH] adding package.xml install rule for soss-ros2-test

---
 CMakeLists.txt | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index a271249..bd8a9c2 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -75,3 +75,9 @@ if(WIN32)
   find_file(MOCKDLL NAMES "soss-mock.dll" PATHS "${soss-mock_DIR}" PATH_SUFFIXES "lib" )
   install (FILES ${MOCKDLL} DESTINATION "${CMAKE_INSTALL_PREFIX}/../bin")
 endif()
+
+install(
+  FILES "${CMAKE_CURRENT_LIST_DIR}/package.xml"
+  DESTINATION share/soss-ros2-test
+  COMPONENT soss-ros2-test
+)
-- 
2.17.1

