From 601c294158422aff5771710ae46a63f12342d201 Mon Sep 17 00:00:00 2001
From: "Marco A. Gutierrez" <marcogg@marcogg.com>
Date: Thu, 8 Oct 2020 02:19:46 +0000
Subject: [PATCH] removing tests untili we vendor catch2

---
 CMakeLists.txt | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index dc29bfa..5ab6c79 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -134,6 +134,3 @@ install(
   COMPONENT soss-core
 )
 
-###############################
-# Add testing subdirectory
-add_subdirectory(test)
-- 
2.25.1

