From a910c798995ff435ae1f356e79b38ddbc4765ffb Mon Sep 17 00:00:00 2001
From: "Marco A. Gutierrez" <marco@openrobotics.org>
Date: Thu, 17 Dec 2020 14:04:48 +0800
Subject: [PATCH] removing tests

Signed-off-by: Marco A. Gutierrez <marco@openrobotics.org>
---
 CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index dc29bfa..b1f5be1 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -136,4 +136,3 @@ install(
 
 ###############################
 # Add testing subdirectory
-add_subdirectory(test)
-- 
2.25.1

